spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/pom.xml

```diff
@@ -67,6 +67,12 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
+        </dependency>
+
```

Comment:
use `spring-cloud-starter` instead ?

Reply:
I see that `spring-cloud-starter` contains dependencies that may not be needed, for example: `spring-security-rsa`, so the minimum dependency package is used here